common/static/common/templates/discussion/new-post.underscore

```diff
@@ -33,7 +33,7 @@
         <label class="field-label">
             <span class="field-label-text"><%- gettext("Title") %></span>
             <div class="field-help" id="field_help_title">
-                <%- gettext("Add a clear and descriptive title to encourage participation.") %>
+                <%- gettext("Add a clear and descriptive title to encourage participation. (required)") %>
```

Comment:
@bjacobel Where we're adding "(Required) after a period, let's use uppercase for the initial "R".